packages/amplify-cli/src/extensions/amplify-helpers/delete-project.ts

```diff
@@ -62,7 +62,7 @@ export async function getConfirmation(context, env?) {
     return {
       proceed: true,
       deleteS3: true,
-      deleteAmpilfyApp: true,
+      deleteAmpilfyApp: !process.env.CLI_DEV_INTERNAL_DISABLE_AMPLIFY_APP_DELETION
```

Comment:
There is a deleteAmpilfyApp: true down below as well - do you need to modify that as well?

Reply:
No need to. Tested already